sarif/testdata/zap.sarif.json

```diff
@@ -176,6 +176,67 @@
                         },
                         "noResponseReceived": true
                     }
+                },
+                 {
+                    "level": "error",
+                    "locations": [
+                        {
+                            "physicalLocation": {
+                                "artifactLocation": {
+                                    "uri": "http:\/\/bodgeit.com:8080"
+                                },
+                                "region": {
+                                    "startLine": 1,
+                                    "snippet": {
+                                        "text": ""
+                                    }
+                                }
+                            },
+                            "properties": {
+                                "attack": "5-2"
+                            }
+                        }
+                    ],
+                    "message": {
+                        "text": "The original page results were successfully replicated using the expression [5-2] as the parameter value\nThe parameter value being modified was stripped from the HTML output for the purposes of the comparison."
```

Comment:
if you change the above message in the tests like I suggested I think you will need to update it here as well